components/callbacks.py

```diff
@@ -91,10 +83,10 @@ def inlinequery_help(update: Update, context: CallbackContext) -> None:
         f"Some wiki pages have spaces in them. Please replace such spaces with underscores. "
         f"The bot will automatically change them back desired space."
     )
-    context.bot.send_message(chat_id, text, parse_mode=ParseMode.MARKDOWN)
+    await context.bot.send_message(chat_id, text, parse_mode=ParseMode.MARKDOWN)
```

Comment:
```suggestion
    await update.message.reply_markdown(text)
```
and we can remove `chat_id` set above